sdk/core/Azure.Core.Experimental/src/RequestOptions.cs

```diff
@@ -91,7 +47,7 @@ public void AddClassifier(Func<HttpMessage, ResponseClassification?> classifier)
         /// <summary>
         /// Controls under what conditions the operation raises an exception if the underlying response indicates a failure.
         /// </summary>
-        public ResponseStatusOption StatusOption { get; set; } = ResponseStatusOption.Default;
+        public ResponseStatusOption StatusOption { get; set; } = ResponseStatusOption.ThrowOnError;
```

Comment:
I was confused about whether `Default` meant default LLC behavior or default HLC behavior, so I tried to clarify this.

Reply:
I've removed this to have this conversation later.